float64 pan
float64 tilt
---
float64 new_pan
float64 new_tilt
